internal/fsutil/create.go

```diff
@@ -67,5 +67,19 @@ func createSymlink(o *CreateOptions) error {
 	if err != nil && !os.IsExist(err) {
 		return err
 	}
-	return os.Symlink(o.Link, o.Path)
+	// check if symlink exists, remove if it does and points to something else
+	fileinfo, err := os.Lstat(o.Path)
+	if err == nil && (fileinfo.Mode()&os.ModeSymlink) == os.ModeSymlink {
+		link, err := os.Readlink(o.Path)
+		if err == nil && link != o.Link {
+			err = os.Remove(o.Path)
```

Comment:
You don't need to remove existing symlink if it points to `o.Link` already.

Reply:
Yeah, that was a silly one. I fixed that in the last commit.